dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -1197,20 +1194,7 @@ def new_level
     assert_response :success
   end
 
-  test 'milestone with student not in experiment does not trigger rubric eval job' do
```

Comment:
please restore this test case, and change it to cover the case where a teacher is unverified